### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by efrapp**
For code review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+                <li><svg viewBox="0 0 15 9" class="main-icons" width="15" height="15" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M.906 8.418V0L5.64 4.76.906 8.419zm13 0L9.174 4.761 13.906 0v8.418zM7.407 6.539l-1.13-1.137L.907 9h13l-5.37-3.598-1.13 1.137zM1.297 0h12.22l-6.11 5.095L1.297 0z" fill="#000"></path></svg></li>
+                <li><svg class="main-icons" viewBox="0 0 16 15" width="16" height="15" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.406 5.359L8.978 0v3.215C3.82 3.215.406 8.107.406 12.66 1.653 9.133 4.29 7.517 8.978 7.517v3.2l6.428-5.358z" fill="#000"></path></svg></li>
+                <li><svg stroke="none" width="15" height="30" viewBox="0 0 16 25" class=""><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g stroke="#666" fill="none"><path fill="none" d="M0.5,0.5 L0.5,24.5 L8.1716264,19.5137775 L15.5,24.4404006 L15.5,0.5 L0.5,0.5 Z"></path></g></svg></li>
```
Comment (heyapricot) [inline index.html]: End tag `svg` did not match the name of the current open element `g`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+          <p>Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part of the team, said, “This is huge, as big as it gets.”</p>
+          <p>He continued, “This is a signal from the very earliest universe, sending a telegram encoded in gravitational waves.”</p>
+          <p>The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.</p>
+          <p>Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
+          <div class="story">
+            <iframe src="https://www.nytimes.com/interactive/2019/admin/100000002772592.embedded.html?" class="css-sqdgy8" frameborder="0" scrolling="no" width="100%" height="824"></iframe>
```
Comment (heyapricot) [inline index.html]: The width property of the iframe is set on pixels. The `%` is not used

Comment (heyapricot) [reply index.html]: The frameborder attribute on the iframe element is obsolete. Use CSS instead.

Comment (heyapricot) [reply index.html]: The scrolling attribute on the iframe element is obsolete. Use CSS instead.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+              <li> <a href="#"> John Flanagan, G.O.P. State Senate Leader, Seeks Treatment for Alcohol Abuse </a> </li>
+            </ul>
+         </div>
+      </div>
+      <div class="thumbs-articles">
+        <div class="thumbnails">
+            <figure>
+              <div class="thumbs-img">
+                <img src="img/ultima.jpg" alt="ultima">
+                <figcaption class="thumb_captions"> NASA, via Associated Press </figcaption>
```
Comment (heyapricot) [inline index.html]: Element figcaption not allowed as child of element div in this context. (Suppressing further errors from this subtree.)

Comment (heyapricot) [reply index.html]: This happens multiple times. Please check all the occurrences

File: style.css [hunk 4]
```diff
@@ -0,0 +1,548 @@
...
+    -webkit-letter-spacing: 0.05em;
+    -moz-letter-spacing: 0.05em;
+    -ms-letter-spacing: 0.05em;
+    letter-spacing: 0.05em;
+    line-height: 11px;
+    padding: 8px 9px 6px;
+    text-transform: uppercase;
+}
+
+.header_link,.title,.hrb{
```
Comment (heyapricot) [inline style.css]: Empty block! 👎

File: style.css [hunk 5]
```diff
@@ -0,0 +1,548 @@
...
+}
+
+/* Content section */
+.container{
+
+}
```
Comment (heyapricot) [inline style.css]: Empty block! 👎

File: style.css [hunk 6]
```diff
@@ -0,0 +1,548 @@
...
+    font-family: nyt-cheltenham,georgia,'times new roman',times,serif;
+    font-weight: 700;
+    font-style: italic;
+    font-size: 1.8125rem;
+    line-height: 2.125rem;
+    margin: 0 20px 1rem;
+    position: relative;
+    width: 376px;
+    max-width: 600px;
+    margin: 0 auto;
```
Comment (heyapricot) [inline style.css:112]: Property `margin` is overwritten

File: style.css [hunk 7]
```diff
@@ -0,0 +1,548 @@
...
+/*Video Content*/
+.video-content{
+  width: 100%;
+  height: 357.5px;
+  margin-bottom: 10px;
+}
+.figure figcaption{
+  color: #666;
+  font-family: nyt-imperial,georgia,'times new roman',times,serif;
+  margin: 5px 0px 50px 0;
```
Comment (heyapricot) [inline style.css:210]: You can omit the use of `px` when value is 0

File: style.css [hunk 8]
```diff
@@ -0,0 +1,548 @@
...
+    width: 68%;
+    height: 26px;
+    text-align: center;
+    position: relative;
+    top: 10px;
+}
+.title svg {
+  width: 195px;
+}
+.header_link{
```
Comment (heyapricot) [inline style.css]: Convention is to use [kebab case ](http://wiki.c2.com/?KebabCase)

File: index.html [hunk 9]
```diff
@@ -0,0 +1,376 @@
...
+                <li><svg viewBox="0 0 15 9" class="main-icons" width="15" height="15" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M.906 8.418V0L5.64 4.76.906 8.419zm13 0L9.174 4.761 13.906 0v8.418zM7.407 6.539l-1.13-1.137L.907 9h13l-5.37-3.598-1.13 1.137zM1.297 0h12.22l-6.11 5.095L1.297 0z" fill="#000"></path></svg></li>
+                <li><svg class="main-icons" viewBox="0 0 16 15" width="16" height="15" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.406 5.359L8.978 0v3.215C3.82 3.215.406 8.107.406 12.66 1.653 9.133 4.29 7.517 8.978 7.517v3.2l6.428-5.358z" fill="#000"></path></svg></li>
+                <li><svg stroke="none" width="15" height="30" viewBox="0 0 16 25" class=""><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g stroke="#666" fill="none"><path fill="none" d="M0.5,0.5 L0.5,24.5 L8.1716264,19.5137775 L15.5,24.4404006 L15.5,0.5 L0.5,0.5 Z"></path></g></svg></li>
+                <li><img src="img/message.png" width="30" height="30"></li>
```
Comment (heyapricot) [inline index.html]: Looks a bit different than the rest

File: index.html [hunk 10]
```diff
@@ -0,0 +1,376 @@
...
+          <div class="story">
+            <iframe src="https://www.nytimes.com/interactive/2019/admin/100000002772592.embedded.html?" class="css-sqdgy8" frameborder="0" scrolling="no" width="100%" height="824"></iframe>
+          </div>
+          <p> Dr. Kovac said the chance that the results were a fluke was only one in 10 million. </p>
+          <p> Dr. Guth, now 67, pronounced himself “bowled over,” saying he had not expected such a definite confirmation in his lifetime. </p>
+          <p> "With nature, you have to be lucky,” he said. “Apparently we have been lucky." </p>
+          <p> The results are the closely guarded distillation of three years’ worth of observations and analysis. Eschewing email for fear of a leak, Dr. Kovac personally delivered drafts of his work to a select few, meeting with Dr. Guth, who is now a professor at Massachusetts Institute of Technology (as is his son, Larry, who was sleeping that night in 1979), in his office last week. </p>
+
+          <div class="advertise"> </div>
```
Comment (heyapricot) [inline index.html]: I appreciate what you tried to do here but the page would look better if you remove it

File: index.html [hunk 11]
```diff
@@ -0,0 +1,376 @@
...
+          <p>Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered whether it could ever be proved.</p>
+          <p>If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time and space and energy to science and speculation.</p>
+          <p>Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent, architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless number of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.</p>
+          <strong>‘As Big as It Gets’</strong>
```
Comment (heyapricot) [inline index.html:92]: Is not showing as bold text. Weird

File: index.html (hunk 2 above)
Comment (heyapricot) [inline index.html]: Did you copy the iframe from the original page?

**Review by heyapricot — APPROVED** [review]
#  Well done, the project looks good 👍 
Please address the comments and after that, you are ready to move on. No need from my side for further review.
I appreciate the attention to detail you put by trying to add the gray bars and the icons for the subscribe section.
Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

File: style.css (hunk 6 above)
Comment (efrapp) [reply style.css:112]: I removed the first margin

File: style.css (hunk 7 above)
Comment (efrapp) [reply style.css:210]: px removed from 0

**Pull request merged**